ctru-rs/examples/linear-memory.rs

```diff
@@ -0,0 +1,46 @@
+#![feature(allocator_api)]
```

Comment:
It might be worth mentioning in the doc comment that users also need to opt-in to this feature to use the allocator.